src/Checkout/Cart/Validation/CartValidator.php

```diff
@@ -66,12 +66,14 @@ public function validate(Cart $cart, ErrorCollection $errors, SalesChannelContex
         try {
             $this->settingsValidationService->validate($context->getSalesChannelId());
         } catch (PayPalSettingsInvalidException $e) {
+            // @deprecated tag:v6.8.0 - The parameter order will change in v6.8.0
```

Comment:
Rather, we got two new optional parameters which will be required in v6.8 - I think we can add them now, the extra ID parameter should be ignored in <6.7.4.0